src/output.cpp

```diff
@@ -134,13 +134,13 @@ lame_t airlame_init(mix_modes mixmode, int highpass, int lowpass) {
 		return NULL;
 	}
 
-	lame_set_in_samplerate(lame, WAVE_RATE);
-	lame_set_VBR(lame, vbr_mtrh);
-	lame_set_brate(lame, 16);
-	lame_set_quality(lame, 7);
-	lame_set_lowpassfreq(lame, lowpass);
-	lame_set_highpassfreq(lame, highpass);
-	lame_set_out_samplerate(lame, MP3_RATE);
+    lame_set_in_samplerate(lame, WAVE_RATE);
+    lame_set_VBR(lame, vbr_off);
+    lame_set_brate(lame, 16);
+    lame_set_quality(lame, 4);
+    lame_set_lowpassfreq(lame, lowpass);
+    lame_set_highpassfreq(lame, highpass);
+    lame_set_out_samplerate(lame, MP3_RATE);
```

Comment:
How about changing this to use `WAVE_RATE` and deleting `MP3_RATE`?

`WAVE_RATE` is the actual audio sampling rate (either 8k or 16k depending on if NFM is enabled or not), any value larger than that isn't preserving anything, just adding to the file size.